File: bin/umap_get.py
import pandas as pd
import umap
import os

def umap_process(covid_dist):

    u = umap.UMAP(metric = "precomputed", random_state = 2020)
    covid_umap = u.fit_transform(covid_dist)
    covid_umap_df = pd.DataFrame(data = covid_umap)
    return(covid_umap_df)

File: bin/umap_get.R
library(uwot)

umap_process <- function(covid_dist) {
  
  covid_dist <- dist(covid_dist)
  set.seed(2020)
  covid_umap <- uwot::umap(covid_dist, init = "spectral", metric = "cosine", n_neighbors = 50, min_dist = 0.001, spread = 30, local_connectivity = 10)
  covid_umap_df <- as.data.frame(covid_umap)
  return(covid_umap_df)
  
}
